src/main/java/com/nitnelave/CreeperHeal/block/ExplodedBlockManager.java

```diff
@@ -193,9 +193,11 @@ public static void processExplosion(List<Block> originalBlockList, Location loca
         for(Block b : event.getProtectedBlocks())
         {
             CreeperBlock cb = CreeperBlock.newBlock(b.getState());
-            assert cb != null;
```

Comment:
That has a very different meaning: You're turning a fundamental logical error into a silent check. I don't agree with this change.

Reply:
Correct me if I'm wrong, but I was under the impression that this will cause an AssertationError to be thrown unchecked if a block in an explosion has already been changed to air. That can happen in the event of a multiblock already having been handled, and should not be a cause for concern.

Reply:
Hmmm... You may be right. And there can also be other plugins at work in parallel. Although so far, this hasn't been an issue. Okay, remove the assert.